webui/react/package.json

```diff
@@ -34,7 +34,7 @@
     "@codemirror/lang-markdown": "^6.1.1",
     "@codemirror/lang-python": "^6.1.2",
     "@codemirror/legacy-modes": "^6.3.2",
-    "@glideapps/glide-data-grid": "^5.2.1",
+    "@hpe.com/glide-data-grid": "^5.2.1",
```

Comment:
i think its nice to have docs how to maintain the repo in confluence

Reply:
great idea

Reply:
added forked entry explanation in our wiki